compiler/optimizer/LocalDeadStoreElimination.cpp

```diff
@@ -1151,8 +1148,7 @@ void TR::LocalDeadStoreElimination::killStoreNodes(TR::Node *node)
          TR::SymbolReference *storeSymRef=storeNode->getSymbolReference();
 
          // TODO: improve by not killing stores that are definitely disjoint
-         // if (node->getSymbolReference()->getUseDefAliases().contains(storeSymRef, comp()))
-         if (usedef_vector[storeSymRef->getReferenceNumber()])
+         if (node->getSymbolReference()->getUseDefAliases().contains(storeSymRef, comp()))
```

Comment:
Since you essentially reversed the usage of sparse bit vector here and went back to the original way of doing the check, I thought I would ask if you know why a sparse bit vector usage was added here, i.e. it would be good to know what was the case where it was thought to be beneficial.